### PR TITLE
Update anti-spam-message-headers.md

### PR DESCRIPTION
**Pull request by msbemba**
Added 6xx to the list of reason

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1479

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/anti-spam-message-headers.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ This table describes the fields and possible values for each email authenticatio
...
+|reason|The reason the composite authentication passed or failed. The value for the reason is made up of three digits: <ul><li>**000**: The message explicitly failed authentication. For example, the message received a DMARC fail with an action of quarantine or reject.</li><li>**001**: The message implicitly failed authentication, and the sending domain did not publish authentication policies. For example, a DMARC policy of p=none.</li><li>**1xx** or **7xx**: The message passed authentication. The second two digits are internal codes used by Office 365.</li><li>**2xx**: The message soft-passed authentication. The second two digits are internal codes used by Office 365.</li><li>**3xx**: The message was not checked for composite authentication.</li><li>**4xx** or **9xx**: The message bypassed composite authentication. The second two digits are internal codes used by Office 365.</li><li>**6xx**: The message failed implicit email authentication, and the sending domain is one of your organization's accepted domains (this is part of self-to-self, or intra-org, spoofing).</li></ul>|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/anti-spam-message-headers.md]:
```suggestion
|reason|The reason the composite authentication passed or failed. The value for the reason is made up of three digits: <ul><li>**000**: The message explicitly failed authentication. For example, the message received a DMARC fail with an action of quarantine or reject.</li><li>**001**: The message implicitly failed authentication, and the sending domain did not publish authentication policies. For example, a DMARC policy of p=none.</li><li>**1xx** or **7xx**: The message passed authentication. The second two digits are internal codes used by Office 365.</li><li>**2xx**: The message soft-passed authentication. The second two digits are internal codes used by Office 365.</li><li>**3xx**: The message was not checked for composite authentication.</li><li>**4xx** or **9xx**: The message bypassed composite authentication. The second two digits are internal codes used by Office 365.</li><li>**6xx**: The message failed implicit email authentication and the sending domain is one of your organization's accepted domains (this is part of self-to-self or intra-org spoofing).</li></ul>|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #1479.

Thank you.

cc: @kenwith

**Pull request merged**